harvard-lts/jstor_aspace_transformer
# add transform of 1 set to itest

**add XSLT transform of 1 set to itest**
* * *

**JIRA Ticket**: [(link)](https://jira.huit.harvard.edu/browse/SS-189)

# What does this Pull Request do?
We want to test a xslt transform of 1 set for itests; use a test config file to specify 1 set to transform

# How should this be tested?

Deploy to dev (done, trial)
Run ITests
Confirm presence of 8 files in transformed dir on dev at /docker/jstorforum/PROCESSING/transformeded/warren_test

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests? Y

# Interested parties
Tag (@ mention) interested parties
